### PR TITLE
listing-page modal, textarea validation

### PR DESCRIPTION
**Pull request by IoanaRoxanaPoroch**

### CODE REVIEW

File: src/components/common/modal/Modal.js [hunk 1]
```diff
@@ -27,6 +18,8 @@ export const Modal = ({ open }) => {
     }
   });
 
+  console.log(allImages);
```
Comment (petricafinaga) [inline src/components/common/modal/Modal.js:21]: I think we can remove this console  log

File: src/UI/text/Text.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const Text = ({ children, variant, sx, color }) => {
   return (
     <div>
       <Typography
+        className="test"
```
Comment (petricafinaga) [inline src/UI/text/Text.js:9]: Are you sure you need this class? Or maybe it can be renamed to something more suggestive ;)

**Review by dianamiraute — APPROVED** [review]

File: src/components/listingMessage/ListingMessageStyle.js [hunk 3]
```diff
@@ -26,12 +36,18 @@ export default css`
...
-  width: "750px",
-  color: "#D0D5DD",
+  color: "#1C1C1E",
+  borderRadius: "8px",
+  // [appTheme.breakpoints.only("mobile")]: {},
+  // [appTheme.breakpoints.only("tablet")]: {},
+  // [appTheme.breakpoints.only("laptop")]: {},
+  // [appTheme.breakpoints.only("desktop")]: {
+  //   width: "750px",
+  // },
```
Comment (petricafinaga) [inline src/components/listingMessage/ListingMessageStyle.js:50]: If you don't need this commented code, I think you can remove it

**Review by petricafinaga — CHANGES_REQUESTED** [review]

File: src/components/common/modal/Modal.js (hunk 1 above)
Comment (BiancaDianaAlexiuc) [inline src/components/common/modal/Modal.js:21]: Please remove the console log.

**Review by BiancaDianaAlexiuc — APPROVED** [review]

**Review by petricafinaga — APPROVED** [review]

**Pull request merged**